test_api_models.py

```
@@ -0,0 +1,265 @@
+        "docker_compose_path": "/opt/docker-compose.yml",
+        "docker_appdata_path": "/opt/appdata",
+        "monitoring_enabled": True
+    }
+    
+    async with aiohttp.ClientSession() as session:
+        # Create device
+        async with session.post(
+            'http://localhost:9101/api/devices/',
+            json=device_data,
+            headers={'Authorization': 'Bearer your-api-key-for-authentication'}
+        ) as response:
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)